benchexec/tablegenerator/react-table/src/components/ReactTable.js

```diff
@@ -415,6 +415,15 @@ const TableRender = (props) => {
     return out;
   };
 
+  const addTypeToFilter = (filters) =>
```

Comment:
Is the code duplication really needed? And why does this method some differently for `id` filters?

Reply:
I've put the function in the `Overview` component and unified the handling for `id` filters, i.e. column types are only added to non-`id` filters.